### PR TITLE
Third task commit. Waiting for review.

### PR DESCRIPTION
**Pull request by Radu-Stroe**

### CODE REVIEW

File: src/test/java/internship/framework/pages/Homepage.java [hunk 1]
```diff
@@ -21,6 +20,9 @@
     @FindBy(xpath = "//ul[contains(@class,'header-nav-global__list header-nav-global__engagement')]//li[3]//a[1]")
     private WebElement registerButton;
 
+    @FindBy(css = "a[title='My Account']")
+    private WebElement myAccountButton;
```
Comment (cristinanedin) [inline src/test/java/internship/framework/pages/Homepage.java]: this is duplicated in LoginPage:
 @FindBy(xpath = "//a[@title='My Account']")
    private WebElement myAccountLabel;
either move it there or move that one here.

File: src/test/java/internship/tests/MyNewslettersTest.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import org.testng.asserts.SoftAssert;
+
+public class MyNewslettersTest extends TestBase {
+    private final ThreadLocal<SoftAssert> softAssert = new ThreadLocal<>();
+
+    @Test(testName = "My Newsletters Test",
+            description = "Testing My Newsletters functionality")
+    public void TestingMyNewslettersFunctionality() {
+        softAssert.set(new SoftAssert());
+        new LoginPageTest().checkLoginPage();
```
Comment (cristinanedin) [inline src/test/java/internship/tests/MyNewslettersTest.java]: it is not a good practice to use other test's method

**Review by cristinanedin — CHANGES_REQUESTED** [review]
Please address code review changes!

File: src/test/java/internship/framework/pages/Homepage.java (hunk 1 above)
Comment (Radu-Stroe) [reply src/test/java/internship/framework/pages/Homepage.java]: Fixed.

File: src/test/java/internship/tests/MyNewslettersTest.java (hunk 2 above)
Comment (Radu-Stroe) [reply src/test/java/internship/tests/MyNewslettersTest.java]: Fixed.

**Review by cristinanedin — APPROVED** [review]

**Pull request merged**